boards/xtensa/odroid_go/odroid_go.dts

```diff
@@ -5,7 +5,7 @@
  */
 /dts-v1/;
 
-#include <espressif/esp32/esp32_wrover_e_n4r8.dtsi>
+#include "espressif/esp32/esp32_wrover_e_n16r2.dtsi"
```

Comment:
Is it a fixup of previous commit? Which SiP is the right one?
Also: please leave `<>` instead of `""`.

Reply:
Thanks for the finding! Actually the `odroig_go` uses 4MB os PSRAM so I've added a new dtsi.